hpOneView/resources/networking/uplink_sets.py

```diff
@@ -47,120 +48,16 @@ class UplinkSets(object):
     }
 
-    def __init__(self, con):
-        self._connection = con
-        self._client = ResourceClient(con, self.URI)
-        self._ethernet_network = EthernetNetworks(con)
+    def __init__(self, connection, data=None):
+        super(UplinkSets, self).__init__(connection, data)
+        self._ethernet_networks = EthernetNetworks(connection)
 
-    def get_all(self, start=0, count=-1, filter='', sort=''):
-        """
-        Gets a paginated list of uplink sets based on optional sorting and filtering and is constrained by start and
-        count parameters.
-
-        Filters can be used in the URL to control the number of uplink sets that are returned.
-        With no filters specified, the API returns all uplink sets.
-
-        Args:
-            start:
-                The first item to return, using 0-based indexing.
-                If not specified, the default is 0 - start with the first available item.
-            count:
-                The number of resources to return. A count of -1 requests all items.
-                The actual number of items in the response might differ from the requested
-                count if the sum of start and count exceeds the total number of items.
-            filter (list or str):
-                A general filter/query string to narrow the list of items returned. The
-                default is no filter; all resources are returned.
-            sort:
-                The sort order of the returned data set. By default, the sort order is based
-                on create time with the oldest entry first.
-
-        Returns:
-            list: A list of uplink sets.
-        """
-        return self._client.get_all(start, count, filter=filter, sort=sort)
-
-    def get(self, id_or_uri):
-        """
-        Gets an uplink set with the specified ID.
-
-        Args:
-            id_or_uri: Can be either the uplink set id or the uplink set uri.
-
-        Returns:
-            dict: The uplink set.
-        """
-        return self._client.get(id_or_uri)
-
-    def get_by(self, field, value):
-        """
-        Gets all uplink sets that match the filter.
-
-        The search is case-insensitive.
-
-        Args:
-            field: Field name to filter.
-            value: Value to filter.
-
-        Returns:
-            list: Uplink sets
-
-        """
-        return self._client.get_by(field, value)
-
-    def create(self, resource, timeout=-1):
-        """
-        Creates an uplink set.
-
-        Args:
-            resource (dict): Object to create.
-            timeout:
-                Timeout in seconds. Wait for task completion by default. The timeout does not abort the operation in
-                OneView, just stops waiting for its completion.
-
-        Returns:
-            dict: Created resource.
-        """
-        return self._client.create(resource, timeout=timeout, default_values=self.DEFAULT_VALUES)
-
-    def update(self, resource, timeout=-1):
```

Comment:
here we need to add query param: force

Reply:
Update in base class already have a force param.
https://github.com/HewlettPackard/python-hpOneView/blob/master/hpOneView/resources/resource.py

Reply:
yes, but default value is false but in API800 spec, its true

Reply:
ok. I am approving the PR as of now and lets send out an email to OneView team for clarification.

Reply:
Sure, This has to be reported, let's release it now with the default value as False.